### DISCUSSION TITLE
Creating Groups/Bins in TE3

### DISCUSSION THREAD

**Discussion by tlanza89** [TabularEditor/TabularEditor3#534] [🙋♀️TE3 Q&A] [answered]
I've searched but could not find any mention of the ability to **create groups/bins** in Tabular Editor 3. Is this a possibility when connected through XMLA?

Reference: https://docs.microsoft.com/en-us/power-bi/create-reports/desktop-grouping-and-binning

Thanks in advance.

**Comment by otykier** [comment, marked as answer]
Behind the covers, PBI Desktop simply adds a calculated column with a DAX expression that contains the grouping/bin value for each row. Thus, you can achieve the same thing in TE3 just by adding a calc column.
